scripts/docker-compose.yml

```diff
@@ -220,6 +220,7 @@ services:
       - IMAGE_TAG=${IMAGE_TAG}
       - IMAGE_VERSION=${IMAGE_VERSION}
       - UX_APP_TITLE=${UX_APP_TITLE}
+      - UX_APP_INNKEEPER_TITLE=${UX_APP_INNKEEPER_TITLE}
```

Comment:
Can add this one to .env-example under UX_APP_TITLE (line 176) to pull in to docker compose.